AprilSylph/XKit-Rewritten
# Trim Reblogs: Fix one-item detection

### Description


> Minor mistake I made in https://github.com/AprilSylph/XKit-Rewritten/pull/766: Trim Reblogs should allow trimming a post with one trail item and some content, but it shouldn't allow trimming if there's one trail item and no content, since you can't make an empty post.

Resolves #863.

### Testing steps

Enable Trim Reblogs and scroll through your own posts. Posts with two or more visible items (with or without added content) should have trim buttons, while posts with one item (single reblog or original post) should not.
